app/components/channel_list_row/index.tsx

```diff
@@ -19,13 +19,17 @@ type Props = {
     testID?: string;
     selectable?: boolean;
     selected?: boolean;
+    disabled?: boolean;
 }
 
 const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
     return {
         titleContainer: {
             marginLeft: 16,
+            paddingHorizontal: 10,
```

Comment:
This change was needed to make it "selectable", let me know if you have any concern, I've checked in browse channels and the integration selector and the design is good.

Reply:
@matthewbirtch Can you sanity check this?

Reply:
Could you share some screenshots to see the UI? Or is there a better way for me to test this?

Reply:
@javaguirre based on the screen capture here, I'm seeing inconsistencies with the `Find channels` modal and the `Select a channel` UI.

1. In the `Select a channel` UI, there seems to be extra space between the icon and the channel name.
2. This view uses dividers, whereas the `Find channesl` doesn't
3. The search input does not have the same margin as it does in the `Find channels` view, so it's sitting too close to the edges
4. The search input has strange highlight when active

<img width="765" alt="image" src="https://user-images.githubusercontent.com/2040554/216124098-b8880f46-4549-4e83-9e20-93161f945a69.png">

Reply:
Thank you @matthewbirtch! I'll fix them when I have time.